src/components/MapboxMap/index.tsx

```diff
@@ -1,30 +1,36 @@
-import React, { useEffect, useState, useRef, useCallback, useMemo } from "react";
+import {
+  LngLatBoundsLike,
+  LngLatLike,
+  MapOptions,
+  RequestTransformFunction,
+  Map as MbMap,
```

Comment:
Any chance you could call this `MapboxMap`? Or just use the default import and then it can be `mapboxgl.Map`.